homeassistant/components/bang_olufsen/websocket.py

```diff
@@ -92,8 +92,18 @@ def on_notification_notification(
         self, notification: WebsocketNotificationTag
     ) -> None:
         """Send notification dispatch."""
+
         if notification.value:
-            if WebsocketNotification.REMOTE_MENU_CHANGED in notification.value:
+            if notification.value in (
```

Comment:
The library's `WebsocketNotificationTag.value` is partially aligned with the integration's `WebsocketNotification` `StrEnum`.

I think something like this would be more readable:
```py
           ws_notification = try_parse_enum(WebsocketNotification, notification.value)

            if ws_notification in WebsocketNotification.BEOLINK_PEERS,
                WebsocketNotification.BEOLINK_LISTENERS,
                WebsocketNotification.BEOLINK_AVAILABLE_LISTENERS,
            ):
                ...
```